### ISSUE TITLE
Quietening the mybinder badge github action

### ISSUE THREAD

**Issue by manics** [jupyterhub/binder#212]
### Proposed change
This is a follow-up from https://github.com/manics/jupyter-offlinenotebook/pull/54#issuecomment-674766041
> I think bots that update their comment is the way to go instead of posting a new comment each time.

I've been using the mybinder badge github action from https://github.com/jupyterhub/binder/pull/203 (and https://github.com/jupyterhub/binder/pull/206 https://github.com/jupyterhub/binder/pull/208) on some of my repos.. As @betatim pointed out above the action is a bit too chatty as it comments on every push to the PR :smile: 

I written a typescript action that comments once and updates it: https://github.com/manics/action-binderbadge

### Alternative options
Do nothing

### Who would use this feature?
People who want a badge on PRs without repeated notifications of new comments.

### (Optional): Suggest a solution
https://github.com/manics/action-binderbadge
Maybe people could try it out, then later decide whether to recommend it instead of the current action?

@hamelsmu

**Comment by manics** [comment]
E.g. https://github.com/manics/action-binderbadge/pull/9#issuecomment-687364839
![image](https://user-images.githubusercontent.com/1644105/92285520-6ad90c00-eefc-11ea-811f-936d97147222.png)

**Comment by hamelsmu** [comment]
Wow this is so cool @manics !!!

**Comment by hamelsmu** [comment]
@choldgraf @betatim check it out ☝️ I think this is a big improvement!

**Comment by choldgraf** [comment]
oh nice! I feel like if this is an improvement to our current action, the best path forward would be to upstream these improvements to it. What do you think?

**Comment by hamelsmu** [comment]
Dumb question : I don’t think there is a “current” action anywhere to
upstream to, right?  Am I missing something?  Sorry for being slow

**Comment by hamelsmu** [comment]
More clarity - I think what @manics made is a new Action, whereas all we have for binder badges are example GitHub workflow examples?

**Comment by manics** [comment]
Exactly! The current action is a GitHub script that uses `actions/github-script@v1`, whereas this is a standalone action, so upstreaming effectively means transferring the action repo to the jupyterhub org. That's why I thought people should try it out first before we commit to supporting it.

**Comment by choldgraf** [comment]
ahh I see - OK cool I think I had thought we already had an action for some reason , but was mistaken 👍

**Comment by manics** [comment]
@choldgraf Maybe you're thinking of https://github.com/jupyterhub/repo2docker-action ?

**Comment by choldgraf** [comment]
mostly I am just sleep deprived because I've got a 16 day old infant on my hands :-)

**Comment by hamelsmu** [comment]
No worries.  Seems like you are the most productive new father I know!  Also congrats!🎈

**Comment by betatim** [comment]
I like it when bots update their comments instead of spamming a PR with comments :)

But I know nothing about GH Actions vs workflows so I defer to someone else.

On the "trying it out" front: can we enable this for the https://github.com/binder-examples/ repositories?

**Comment by manics** [comment]
I've opened PRs for the 4 pinned binder-example repositories.

I've also got a script that can be used to add it (or any other config file) to other repos...
https://github.com/manics/github-batch-updater
`for r in conda r-conda r requirements; do GITH
UB_TOKEN=$(git config github.token) ./main.js addfile binder-badge.yml --base binder-examples/$r --dest .github/workflows/binder-badge.yml --branch add-binder-badge --title 'Automatically add mybinder badge on PRs' --body $'Add workflow to create a binder badge comment on PRs\n\nSee https://github.com/jupyterhub/binder/issues/212#issuecomment-687706341' --force; done`

**Comment by iamjulia020225** [comment]
Проверим
